test/Java8andUp/playlist.xml

```diff
@@ -1256,7 +1256,7 @@
 	-excludegroups $(DEFAULT_EXCLUDE); \
 	$(TEST_STATUS)</command>
 		<tags>
-			<tag>extended</tag>
+			<tag>sanity</tag>
```

Comment:
To run testNG test in sanity, we need to update the test group annotation in java as well. `@Test(groups = { "level.sanity", "j9vm_SE80" })`

Reply:
Thanks.  Fixed.